scripts/setup/configure-rabbitmq

```diff
@@ -5,11 +5,16 @@
 set -e
 set -x
 
+RABBITMQ_HOST="${RABBITMQ_HOST:-rabbit@$(hostname)}"
```

Comment:
I think this variable should be called `RABBITMQ_NODE`, since technically it's a node name, not a host name (you can have >1 node on a host, for example).